### ISSUE TITLE
windows installer broken

### ISSUE THREAD

**Issue by networks1** [ActiveState/OpenKomodoIDE#16]
MSI throws error dialog: This installation package could not be opened. Contact application vendor to verify that this is a valid Windows installer package.

**Comment by th3coop** [comment]
Can you provide any more information? maybe a screenshot?  or where you got the installer from?  I grabbed it from the Open source repo [here](https://github.com/ActiveState/OpenKomodoIDE/blob/master/installers/Komodo-IDE-12.0.1-91869.msi) and it works

**Comment by networks1** [comment]
Got the same installer. Maye problem was I was trying to I stall on W11?
________________________________

**Comment by networks1** [comment]
Well this is strange. I got the installer from the last link on [this page](https://github.com/ActiveState/OpenKomodoIDE/tree/master/installers). I just retrieved again using the link you provided. It had the same version numbers but it worked. I'm not good enough with Github to tell if your link points to the same or a different file than the one I originally downloaded. If it's the same one, I can't explain this. If it's not the same file, the one I downloaded may be corrupted. In any case my problem is resolved, so thanks for your help.
